app/api/auth.py

```diff
@@ -23,6 +24,10 @@
 from app.models.user import User
 
 auth_routes = Blueprint('auth', __name__, url_prefix='/v1/auth')
+api_key = TwitterOAuth.get_client_id()
```

Comment:
@iamareebjamal these calls which require fetching values from the settings table are being placed before the app is properly initialized I think. If dummy values are passed to the 
`make_twitter_blueprint(api_key=dummmy_api_key, api_secret=dummy_api_secret)`
the build continues, can you confirm this? Also, I think this is not the correct file to register this blueprint, rather should be registered with other blueprints.

Reply:
@CosmicCoder96 How are we going to put in the api key and secret for now?